website/src/content/docs/general/support-policy.md

```diff
@@ -1,39 +1,82 @@
-However the scope of this support is limited to two things:
+This manual contains general information on working with TrueCharts Charts using Helm directly.
+As well as when editing Helm Yaml for tools like FluxCD.
+
+We at TrueCharts offer free support via a support-ticketing system on Discord.
+
+## General Support Policy
+
+- We help out where we can, as long as you tried to do it yourself *first*. We do not provide a walkthrough service.
+- Our staff will try to help out with providing instructions on how to use `kubectl` and other `*ctl` tools where needed.
+- We do not support what we do not build.
+- We do not support any Cloud providers or OpenShift.
+
+## ClusterTool Support Policy
+
+- We offer support for all ClusterTool functions, except those under `adv` and `charts`
+- If you deviate from default configurations significantly, we cannot guarantee a bootstrap will work and support will be limited on a case-by-case basis
+- We do provide support for issues regarding talconfig.yaml, as long as ClusterTool is used for the deployment.
+- We do provide *basic* assistance with Talos OS, when deployed or maintained through ClusterTool (upgrades, config changes etc)
+- We do provide support for FluxCD setup and layout, as long as ClusterTool is used for the deployment.
+
+## Kubernetes Support Policy
+
+- We do not offer support for any component of kubernets that is not made by TrueCharts.
+- This includes charts by other parties or manually created kubernetes manifests and deployments.
+- We do not offer support for any component of kubernets that is not made by TrueCharts.
+
+## Helm Chart Support Policy
+
+- We offer support for all features available in our helm-charts, including all options in the common-chart.
+- We do not support external secret management outside of values.yaml
+
+:::note
 
-- ClusterTool Commands
-- The `values.yaml` of our Helm charts
+Some (common-chart) features might not work correctly with all charts.
+
+:::
+
+## FluxCD Support Policy
+
+- We offer support for our Helm-Charts values in FLuxCD as outlined in Helm Chart Support.
+
+## Structural rules
+
+- If its covered by support as outline above make a ticket in [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826).
+- [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) is not rendered on GitHub or any other platform.
+- Bugs must be reported on GitHub, but only after confirmation by the staff via a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826)
+- Anything not covered by the above Support Policy is not eligble for a [⁠🎫・support ticket](https://discord.com/channels/830763548678291466/936275413179723826). If you need assitance outside the scuoe of [⁠🎫・support](https://discord.com/channels/830763548678291466/936275413179723826) make a thread in an appropiate [Discord Channel](https://discord.gg/tVsPTHWTtr).
 
 ## Expected Prior Knowledge
```

Comment:
Maybe move this below the moved-and-renamed structural rules section.
so:
- tip
- intro
- structural rules
- expected prior